bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php

```diff
@@ -15,6 +15,8 @@
 
 namespace Pimcore\Bundle\AdminBundle\Controller\Admin\Asset;
 
+use DateTimeImmutable;
+use DateTimeZone;
```

Comment:
LGTM, just need to remove these lines, plus `$fileExtension = File::getFileExtension($asset->getFilename());` and merge 11.x to fix and pass tests.

Reply:
Done.